### DISCUSSION TITLE
Welcome to SlideLoader Discussions!

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/SlideLoader#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tani-dubey** [comment]
Hey @birm,
I’m Tanisha Dubey. I just finished setting up the codebase and noticed that setting it up on Windows can be a bit tricky (it requires a WSL environment first) and some steps aren’t very clear for new contributors.

Would it be okay if I update the README to include a short section on setting up SlideLoader locally on Windows/WSL (and maybe Docker), so that newcomers can follow it more easily?

This should make the onboarding smoother and reduce confusion for first-time contributors. 🙂
